File: Arcade/Intro/javascript/electionsWinners.js
function electionsWinners(votes, k) {
    let winners = 0;
    for(let i = 0; i < votes.length; i++) {
        let win = true;
        for(let j = 0; j < votes.length; j++) {
            if(j != i && votes[i] + k <= votes[j]) {
                win = false;
                break;
            }
        }
        if(win) winners++;
    }
    return winners;
}

File: Arcade/Intro/Ruby/electionsWinners.rb
def electionsWinners(votes, k)
    winners = 0
    votes.size.times do |i|
        win = true
        votes.size.times do |j|
            if i != j and votes[i] + k <= votes[j]
                win = !win
                break
            end
        end
        winners += 1 if win
    end
    return winners
end